### PR TITLE
Adding Sidebar to Metrics Home

### PR DESCRIPTION
**Pull request by sejli**
### Description
Adding sidebar to Metrics Home populated with dummy data and visualizations

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/public/components/metrics/helpers/utils.tsx [hunk 1]
```diff
@@ -2,15 +2,19 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
+/* eslint-disable no-console */
```
Comment (ps48) [inline dashboards-observability/public/components/metrics/helpers/utils.tsx:5]: why was this added here?

File: dashboards-observability/public/components/metrics/helpers/utils.tsx [hunk 2]
```diff
@@ -41,3 +45,17 @@ export const onTimeChange = (
...
+  await http
+    .get(`${CUSTOM_PANELS_API_PREFIX}/visualizations/`)
+    .then((res) => {
+      savedVisualizations = res.visualizations;
+    })
+    .catch((err) => {
+      console.error('Issue in fetching all saved visualizations', err);
+    });
+  return savedVisualizations;
+};
```
Comment (ps48) [inline dashboards-observability/public/components/metrics/helpers/utils.tsx:61]: Let's remove this if not needed

File: dashboards-observability/public/components/metrics/helpers/utils.tsx (hunk 1 above)
Comment (sejli) [reply dashboards-observability/public/components/metrics/helpers/utils.tsx:5]: It prevents the error in `console.error()` as shown in [this file](https://github.com/opensearch-project/observability/blob/edbc7cb915a26fbbf590cb2f7e1445a50233a0e8/dashboards-observability/public/components/custom_panels/helpers/utils.tsx#L5) but if we remove the function all together, I can get rid of it.

File: dashboards-observability/public/components/metrics/helpers/utils.tsx (hunk 2 above)
Comment (sejli) [reply dashboards-observability/public/components/metrics/helpers/utils.tsx]: removed

**Review by ps48 — APPROVED** [review]
